add new cache page (#9949)
getsentry/sentry-docs: docs/product/performance/cache-monitoring/cache-page.mdx

---
title: "Cache Page"
sidebar_order: 55
description: "Learn more about Sentry's Cache page where you can get an overview of your cache performance and catch potential cache performance issues. "
---

The **Caches** page gives an overview of cache performance across all endpoints for currently selected backend projects with summary graphs for **Miss Rate** and **Requests Per Minute** (throughput). You can use these as a starting point to see if there are any potential cache performance issues, for example, a higher than expected Miss Rate percentage.

>  If you see an anomaly or want to investigate a time range further, click and drag to select a range directly in the graph and you'll see data for that specific time range only.

Reviewer: ```suggestion
 If you see an anomaly or want to investigate a time range further, click and drag to select a range directly in the graph and data will be filtered for that specific time range only.
```

The transaction table shows a list of endpoints that contain at least one `cache.get` span along with:

- Its average value size (the bytes being fetched from cache)
- Throughput
- Average transaction duration

> - Miss rate percentage (how often did a lookup from cache fail)

Reviewer: ```suggestion
- Miss rate percentage (how often a cache lookup failed)
```

Reviewer 2: Its not really a failed lookup, its a lookup that returned no value
```suggestion
- Miss rate percentage (how often did a lookup did not return a value)
```

- Time spent (total time your application spent on a given transaction)

By default, this table is sorted by most time spent. This means that endpoints at the top are usually really slow, requested very frequently, or both. You can change the sort order to view top transactions by throughput, miss rate percentage, average value size, or average transaction duration.

To investigate a specific endpoint, click on it to open a sidebar showing some sample events and additional duration metrics.

## Sample List

> To be able to compare the performances of cache hits (found a value in the cache) versus misses (failed to find a value in the cache corresponding to the key) over time, Sentry automatically looks for samples of both for the timeframe selected from the **Caches** page.

Reviewer: ```suggestion
To help you compare the performances of cache hits (where a value was found in the cache) versus misses (where no value corresponding to the key was found in the cache) over time, Sentry automatically surfaces a distribution of both samples for the timeframe selected from the **Caches** page.
```

> These sample events are shown as triangles in the average duration graphs above. If you click on a transaction, you'll be able to view this list of samples. The table shows a sample list of spans, their key, transaction duration, and whether or not all cache lookups within the transaction either hit or missed.

Reviewer: ```suggestion
These sample events are shown as triangles in the average duration graphs at the top. These sample events are also listed in the table below, along with their spans, key, transaction duration, and hit/miss status.
```
Not sure what transaction you're referring to here, but I think we can just remove that sentence since we're already talking about the sample list?

Reviewer 2: 👍
